### PR TITLE
[PLT-6668] tests for Tabs component

### PR DESCRIPTION
**Pull request by licarm**
## Where to Start
Tabs.test.jsx

### CODE REVIEW

File: packages/riipen-ui/src/components/Tabs.test.jsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    });
+  });
+
+  describe("component prop", () => {
+    it("applies given component to the root node", () => {
+      const component = "h3";
+
+      const wrapper = mount(<Tabs component={component} />);
+
+      expect(wrapper.find("Tabs").childAt());
```
Comment (imogenem) [inline packages/riipen-ui/src/components/Tabs.test.jsx:142]: Is this supposed to be....

```
     expect(
        wrapper
          .find("Tabs")
          .childAt(0)
          .name()
      ).toEqual(component);
```

File: packages/riipen-ui/src/components/Tabs.test.jsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      const wrapper = mount(
+        <Tabs value={value}>
+          {child1}
+          {child2}
+        </Tabs>
+      );
+
+      const childrenNodes = wrapper.find("Tab");
+      expect(childrenNodes.at(0).props().active).toEqual(false);
+      expect(childrenNodes.at(1).props().active).toEqual(true);
```
Comment (imogenem) [inline packages/riipen-ui/src/components/Tabs.test.jsx:202]: Coooool

File: packages/riipen-ui/src/components/Tabs.test.jsx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+      const wrapper = mount(
+        <Tabs>
+          {child1}
+          {child2}
+        </Tabs>
+      );
+
+      const childrenNodes = wrapper.find("Tab");
+      expect(childrenNodes.at(0).props().value).toEqual("one");
+      expect(childrenNodes.at(1).props().value).toEqual("two");
```
Comment (imogenem) [inline packages/riipen-ui/src/components/Tabs.test.jsx:76]: Is the second test necessary here? Looks like they check the same thing.

**Review by imogenem — COMMENTED** [review]
Nice use of Tab!

File: packages/riipen-ui/src/components/Tabs.test.jsx [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+      const component = "abcd";
+
+      mount(<Tabs component={component} />);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("onChange prop", () => {
+    it("passes valid handler to the onClick prop of children nodes", () => {
```
Comment (jordanell) [inline packages/riipen-ui/src/components/Tabs.test.jsx:156]: You are mixing your tests up here I think. You are saying "check to make sure it's passed to children", but then your test is actually testing the click functionality.

I think you should maybe split this into two tests:
1. Does it pass the prop to the children
2. Does the click functionality work

File: packages/riipen-ui/src/components/Tabs.test.jsx [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+      const wrapper = mount(<Tabs onChange={onChange}>{child}</Tabs>);
+
+      wrapper.find("Tab").simulate("click");
+
+      expect(onChange).toHaveBeenCalledWith(expect.any(Object), "one");
+    });
+  });
+
+  describe("orientation prop", () => {
+    it("passes given valid orientation to children", () => {
```
Comment (jordanell) [inline packages/riipen-ui/src/components/Tabs.test.jsx:169]: I think we should add a second test that checks for the `orientation` prop is also used in the `className` found on line 94.

**Review by jordanell — CHANGES_REQUESTED** [review]

**Review by jordanell — APPROVED** [review]

**Pull request merged**